src/main.c

```diff
@@ -209,12 +209,17 @@ int main(int argc, char* args[])
     {"version", 'v', OPTPARSE_NONE},
     {"record", 'r', OPTPARSE_OPTIONAL},
     {"scale", 's', OPTPARSE_REQUIRED},
+    {"args", 'a', OPTPARSE_OPTIONAL}, // Game exclusive settings
```

Comment:
We probably don't need this if we can provide all the command line flags to the game